8Knot/queries/query_template.py

```diff
@@ -1,22 +1,20 @@
 import logging
 import pandas as pd
-from db_manager.augur_manager import AugurManager
-from app import celery_app
-from cache_manager.cache_manager import CacheManager as cm
+import cache_manager.cache_facade as cf
 import io
-import datetime as dt
-from sqlalchemy.exc import SQLAlchemyError
 
 """
 TODO:
 (1) update QUERY_NAME
 (2) update 'NAME_query' found in function definition and in the function call that sets the 'ack' variable below.
 'NAME' should be the same as QUERY_NAME
-(3) paste SQL query in the query_string
+(3) paste SQL query in the query_string. Make sure it's wild-carded for psycopg2
 (4) insert any necessary df column name or format changed under the pandas column and format updates comment
 (5) reset df index if #4 is performed via "df = df.reset_index(drop=True)"
 (6) go to index/index_callbacks.py and import the NAME_query as a unqiue acronym and add it to the QUERIES list
-(7) delete this list when completed
+(8) specify the correct table name in the call to 'caching_wrapper'
```

Comment:
also is there any reason why table name shouldnt just be the QUERY_NAME

Reply:
> Change "table name" to be a variable at the top to be assigned there and use the variable. Also what is "table name? Not sure what it is to be able to know what to assign to it

This answers both comments- 'table name' is the table in the database, and many queries use the same tables. There's the "commits, issues, prs, etc." tables that are all used by one or more queries.

Reply:
Im still a little confused because many queries use multiple tables

Reply:
Ah I can definitely be more precise-

In 8Knot, the existing tables for caching are "commits, issues, prs, contributors, issue_assignments, pr_assignments" etc. "tablename" here is what table the query executed against Augur should ultimately write to. The response from augur uses multiple tables, such as for issue_assignments, but only writes to one table in cache. 

As for why query_name, func_name, and table_name can't all be the same value:

query_name and func_name could, and likely should, be the same. I'll update that.

table_name (the table in cache) isn't necessarily the same as query_name (the function that initiates the caching flow)- it could be, but the DB schema uses shorter names for tables, like "commits" vs. "commits_query" so it seemed more flexible to just allow the developer to specify the target table rather than commit them to writing to tables that have the same name as the calling function.

Reply:
Personally, I would side to keep table name under the hood/not a part of the query creation and visualization process and stick with having the set of queries to use and get imported an used in visualizations. From my experience and talking with sean about others using 8knot the simplicity of accessing the data was a huge plus. It adds a lot of complexity

Reply:
okay removed this in the last update